src/features/counter/Counter.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
```

Comment:
We don't need this file as of now this was done for testing purpose, so now as we are merging to the beta branch, this counter files should be removed.